packages/search-ui/src/actions/removeFilter.ts

```diff
@@ -9,7 +10,7 @@ import { removeSingleFilterValue } from "../helpers";
  * @param value String (Optional) field value for filter to remove
  * @param type String (Optional) type of filter to remove
  */
-export default function removeFilter(name, value, type) {
+export default function removeFilter(name: string, value: FilterValue, type: FilterType) {
```

Comment:
Suggesting to mark some params as optional based on the comment above.
```suggestion
export default function removeFilter(name: string, value?: FilterValue, type?: FilterType) {
```